front/js/components/main.es6

```diff
@@ -93,23 +88,35 @@
     DOM.$reviewsNavItems
       .removeClass('active')
       .eq(reviewID).addClass('active');
-  };
+  }
 
-  function toggleSearchBar() {
-    if (!DOM.$searchBar.length) return;
+  /**
+   * Move searchbar on index page.
+   * This logic is required cause of banner on index page.
+   */
+  function moveSearchBar() {
```

Comment:
We have a baner at the header index page. After scrolling to the height of the banner, we fix search field below navigation block like on all other pages.